sklearn/decomposition/tests/test_kernel_pca.py

```diff
@@ -286,16 +286,16 @@ def test_kernel_conditioning():
     assert np.all(kpca.lambdas_ == _check_psd_eigenvalues(kpca.lambdas_))
 
 
-@pytest.mark.parametrize("kernel",
-                         ["linear", "poly", "rbf", "sigmoid", "cosine"])
-def test_kernel_pca_inverse_transform(kernel):
-    X, *_ = make_blobs(n_samples=100, n_features=4, centers=[[1, 1, 1, 1]],
-                       random_state=0)
-
-    kp = KernelPCA(n_components=2, kernel=kernel, fit_inverse_transform=True)
-    X_trans = kp.fit_transform(X)
-    X_inv = kp.inverse_transform(X_trans)
-    assert_allclose(X, X_inv)
+def test_kernel_pca_inverse_transform_reconstruction():
+    # Test if the reconstruction is a good approximation.
```

Comment:
```suggestion
    # Test if the reconstruction is a good approximation.
    # Note that in general it is not possible to get an arbitrarily good
    # reconstruction because of kernel centering that does not
    # preserve all the information of the original data.
```